extensions/wikidata/src/org/openrefine/wikidata/qa/WikidataConstraintFetcher.java

```diff
@@ -340,4 +333,47 @@ public boolean usableOnItems(PropertyIdValue pid) {
         }
         return results;
     }
+
+    /**
+     * Returns the list of PropertyIdValues of conflicting statements
+     *
+     * @param pid:
+     *            the property having conflicts-with constraint
+     * @return
+     */
+    public List<Value> getConflictsWithProperties (PropertyIdValue pid){
```

Comment:
Style: no space between the method name and the opening parenthesis, one space between the closing parenthesis and the opening curly brace.

Reply:
Okay, I'll do the changes.